### PR TITLE
Feature/kafka msk unsplit

### PR DESCRIPTION
**Pull request by gedge**
### What

make this app able to use TLS on all topics
(was using legacy - non-TLS - kafka for cantabular topic before this)

should not be merged until cantabular apps are also ready for TLS

### How to review

ensure all removed

### Who can review

!gedge

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -76,14 +74,12 @@ func Get() (*Configuration, error) {
 		DefaultMaxLimit:            1000,
 		DefaultOffset:              0,
 		KafkaConfig: KafkaConfig{
+			Brokers:                               brokers,
```
Comment (justinpjose) [inline config/config.go:77]: Why is this separated out? It's convenient for the actual default value to be mentioned here to be consistent with all the other configs

Comment (justinpjose) [reply config/config.go:77]:
```suggestion
			Brokers:                               []string{"localhost:9092"},
```
Add remove line 58

File: go.mod [hunk 2]
```diff
@@ -2,26 +2,36 @@ module github.com/ONSdigital/dp-import-api
 
 go 1.17
 
+replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
```
Comment (justinpjose) [inline go.mod:5]: Is it worthwhile to add a comment above on why we are replacing?

Comment (gedge) [reply go.mod:5]: good idea

**Pull request merged**